### ISSUE TITLE
bug(ansible-semaphore): Database secret not generated

### ISSUE THREAD

**Issue by mlinares1998** [cloudhippie/charts#114]
Hi! 
I'm testing the chart to deploy Semaphore with ArgoCD + external PostgreSQL database.
The deploy is failing due missing ansible-semaphore-database secret which is expected by the deployment with this configuration.

![image](https://github.com/cloudhippie/charts/assets/15200008/2fef0247-b790-4758-b4de-fcffa67d2496)
![image](https://github.com/cloudhippie/charts/assets/15200008/6267b3cf-6c7e-4a72-8794-21ed6da7a921)

I have checked the chart and the required template for the database in the secret.yaml file is missing.
Adding the following snippet in secret.yaml template fixes the issue.

```go
{{- if and (not .Values.database.existingSecret) (not (eq .Values.database.type "bolt" )) }}
---
apiVersion: v1
kind: Secret

metadata:
  name: {{ include "ansible-semaphore.fullname" . }}-database
  namespace: {{ .Release.Namespace }}
  labels:
    {{- include "ansible-semaphore.labels" . | nindent 4 }}
{{- if .Values.labels }}
    {{- toYaml .Values.labels | nindent 4 }}
{{- end }}
{{- if  .Values.annotations }}
  annotations:
    {{- toYaml .Values.annotations | nindent 4 }}
{{- end }}

type: Opaque
data:
  {{ .Values.database.usernameKey }}: {{ .Values.database.username | b64enc }}
  {{ .Values.database.passwordKey }}: {{ .Values.database.password | b64enc }}
{{- end }}
```

**Comment by tboerger** [comment]
Damn, you are right, looks like I have overwritten this somehow.

**Comment by tboerger** [comment]
Sorry, I just forgot about this issue. I have released v6.5.0 which includes this fix.
